docs/source/intro/installation.rst

```diff
@@ -1,16 +1,892 @@
-.. _intro/install_advanced:
+.. _intro:install:
 
 *********************
-Advanced Installation
+Detailed Installation
 *********************
 
-`#4021 <https://github.com/aiidateam/aiida-core/issues/4021>`_
+.. _intro:install:prerequisites:
 
-- All info from current `docs/source/install/`
-- Pip + Link to how to install services
-- In-depth/alternative installation methods
-- Services for various systems
-- Conda in detail
-- Pip in detail
-- QuantumMobile
-- AiiDA lab
+Installing Prerequisites
+========================
+
+AiiDA is designed to run on `Unix <https://en.wikipedia.org/wiki/Unix>`_ operating systems and requires the following software:
+
+* `bash <https://en.wikipedia.org/wiki/Bash_(Unix_shell)>`_ or
+  `zsh <https://en.wikipedia.org/wiki/Z_shell>`_ (The shell)
+* `python`_ >= 3.6 (The programming language used by AiiDA)
+* `python3-pip`_ (Python 3 package manager)
+* `postgresql`_ (Database software, version 9.4 or higher)
+* `RabbitMQ`_ (A message broker necessary for AiiDA to communicate between processes)
+
+Depending on your set up, there are a few optional dependencies:
+
+* `git`_ (Version control system used for AiiDA development)
+* `graphviz`_ (For plotting AiiDA provenance graphs)
+
+.. _graphviz: https://www.graphviz.org/download
+.. _git: https://git-scm.com/downloads
+.. _python: https://www.python.org/downloads
+.. _python3-pip: https://packaging.python.org/installing/#requirements-for-installing-packages
+.. _virtualenv: https://packages.ubuntu.com/bionic/virtualenv
+.. _virtualenvwrapper: https://packages.ubuntu.com/bionic/virtualenvwrapper
+.. _postgresql: https://www.postgresql.org/downloads
+.. _RabbitMQ: https://www.rabbitmq.com/
+
+AiiDA has been tested on the following platforms:
+
+* Ubuntu 14.04, 16.04, 18.04
+* Mac OS X
```

Comment:
@sphuber Can we qualify this further?

Reply:
Not sure, to be honest. Depends on how how strict we want to have "tested" mean. If that should only include with what we have tested through CI, we should technically only list 18.04. If we want to be a bit more lenient, we could add 16.04 and Mac OS X, as we know people how have run successfully on it or are currently running on it. However, this might be a slippery slope, because for what versions of AiiDA does this hold? When do we remove a platform if it has not been tested in a long time. Do we add WSL, since we have some people that have run on it? This for example I don't think we should officially mention.

Reply:
Well we are mentioning WSL pretty extensively as a "supported installation method". I think we should have a brief team discussion about this at the next meeting and then we can make sure to consistently qualify what is considered a "supported system".